homework_6.py

```diff
@@ -28,10 +28,10 @@
 # Напечатайте текст: “Привет, Ivan Ivanou!
 # Добро пожаловать в Minsk Belarus”
 fullname_in_list = ["Ivan", "Ivanou"]
-city_in_str = "Minsk"
-country_in_str = "Belarus"
-print("Привет,", ' '.join(fullname_in_list) + "!", "Добро пожаловать в", city_in_str,
-      country_in_str)
+CITY_IN_STR = "Minsk"
+COUNTRY_IN_STR = "Belarus"
+print("Привет,", ' '.join(fullname_in_list) + "!", "Добро пожаловать в",
+      CITY_IN_STR, COUNTRY_IN_STR)
 
 # Дан список ["I", "love", "arrays", "they", "are", "my", "favorite"]
 # сделайте из него строку => "I love arrays they are my favorite"
```

Comment:
if it's possible add one more solution with for loop and string concatenation 
something like: result_str = ''
for word in list: 
bla bla bla
print(result_str)

Reply:
I found how to do it on the internet, but it's still very difficult for me.